docker-compose.test.yaml

```diff
@@ -16,4 +16,10 @@ services:
     environment:
       DDI_API_BASE_URL: test
       DDI_EVENTS_BASE_URL: test-events
-      NODE_OPTIONS: "--max-old-space-size=3072"
\ No newline at end of file
+      NODE_OPTIONS: "--max-old-space-size=3072"
+      # This is a fake private key used purely for testing
+      JWT_PRIVATE_KEY: "LS0tLS1CRUdJTiBQUklWQVRFIEtFWS0tLS0tCk1JSUV2QUlCQURBTkJna3Foa2lHOXcwQkFRRUZBQVNDQktZd2dnU2lBZ0VBQW9JQkFRQ3BwS2Q3TUVQOXBzL3UKQ0hRN1o0YWxlRUxmdXhiZ3BqRFRoSVVkTjBTYlJCeUFoUytaVXlMaStLeDR0cXI2aDVXRmFEN2l0d0IwM2ZMaQpWUHI5S1E1RUhYZVNOSnI3MnBqbndaV2xPWDNSVTNrQnNBbnl6bDlodGQ1Qkx2RkRucXIramRzMTF1VlVHMjdGCjlsRWJ6cHRlRUNEYW1oZXR5ZUhTN2o1MGo5Z1lrSjhvQkk2WDlqdXI0MWJrUEM4YUg0NnVMSis2TjJYYkJLcXIKMzZLWGdLSFppMzR0RzNEcStzQTNKUkVtRDcySFJEUDAxNUs2R3RERFNuSGNwdWFkUDVGeTUzZW1IUjFjMzg1awpZSCtIaUErV25PZnhwcUFxaGxUSnQ3ZWxqdTRpRVJEaTBKZEtmWGxWNDBiUVdPWVdjdXdlVGYyUmYzWnQ2ZUsvCmZ0OXYxcWovQWdNQkFBRUNnZ0VBWTN0TE85SnlESzFtQWhDekdLTDlNc3Z6YTRBclNsSXJEOThHc0x0WVlEVHgKQWUwWW5MRkVKQlh3RGxvUXFob1lkTU5OSFdaenZoVHAraEFibjVvVDQ1M2tJTGo0cWZqbThnRno0QkIyMStvNwphK0h2ODVOU0c5cXJHMDZMN1R0K3l3Z2xKTTJBYm5JbjQ2OHVvVC9ubktHWW9wajZpYzk1dmxnNEIrTFN0cFFqCjJqcnZYbWh5ZTkrSXgzcGhWS0FET2FMZStnU3FtNzVRelg4MjZQZXNwSCtTS1g3N1h5K2Qva2p3SEk2V1R6NHUKQnFiUlF0dDZzVGZ6TytxNHgrOWN1NlBSOWljcnpLcERRVkNtZTF1ZlpIVEp1aUxvZEJDcnp1STdsdks3eG5pZQpzdzR4cWg0ZGFHd0J0VXFZS0lZQ2VwVDY0TlpYMG43L1I2RXVDTW9jQVFLQmdRRGQ4dFB4OGNqVDk1b2UzN3M1CldWTHhpbUlYRWhmRko3SEZ4OC9aMHUrVlQ1dTFwZVlOK1dDQUhMRENXR0I0QnlOM1R5M3oySnkwM0Q0UnZETnEKcERaUkxNRmJHUHFZNC9pb1k5aVRRU3BRT1VLSlVFaUVsTEI2OUZlU3pVSVNLNlZvMzUrSW1FK1lJc2wwOTVLUgpwUzY5KzNNenNPUitIMzRVaWNGRnpHTUpEd0tCZ1FERHEzL1hoa0l1akx0d2RRZG5EVkF2YUN4cjVzUmR4RkVhCjlDZWcybXlySnZONEc2bnpReXNBZllhTEZ3VFZtN09lajFzRklaRlBNRC90OU1TUENGUkFNMk15VlZWeTNJTkoKR0h3bHZ6WWlzZGRseHpta0VLaXBxY3hiZFV6eW1qT1AyUjB6MWhqd3QyaG9IT2NROVRqUFJ5WGp2SzR6N2xsYgpEOHU4dmlZQkVRS0JnQldqOVpKamxzZEVIeHpmaDZ2Qno5cUpPYU15T21zWXdiYWRFdUVVTmhRQzNSWExCVEZiCm5WTWdlUU9hQmwzdHBFR2YwTVV6MjZPdXpjYUE2U3ArdEdVWnpnaUoyUFZMVlJWbUZ0blV3c0ZyczljYm90WjYKZ1BLY0hHWE42YUZIN0NyYmt2eUlqVW95S2g1amZVRUNNQnFLQitPRVhlTUdRdzdSbDZ5c2p3TjFBb0dBWGl4TwpiVGFLNThpSDlnbjlOQlh1UDhMTmpta1VuV1I4L0pJQitLTG5wa0FxOGVXdW1Bdjl5RkNoenBPWG1nOUVGYzQ5CkttMExlS2RxUFdhT3E2Q0FuQjJmWW53N2tmcWJuVFNSbThFNWx0MHRKQ2tNanZBTm5RckI5a3F3YkJyWE1pWjIKclpvZmlzdGJEckRLYlJqNDdKbjduRzRWRVoxbm1ESTBCblVsNldFQ2dZQm5RbmNZSDJDUGg1R2xxOHdzTFQ3TwpoeHoxWU1GNGdUOWltUmR6TEN6RGFzTGxMaXhXMnZWVEhEdVdvMW1HSnZ3NjJkaTI1YUFDRFJZN1cxcmRHN1BNCnRnTnlYK0puRkhKTmp3NGd4dDJtQkJLT1U3OVo0WHU4L1BFdUtURUpjVWh5WnNkS2lIa3hvNW5Eakt0WEJkRDkKOXgzUmRkNXFPMGFNL2lxNFVqNFpCZz09Ci0tLS0tRU5EIFBSSVZBVEUgS0VZLS0tLS0="
```

Comment:
Are we 100% sure these are dummy keys and not copied from your own OneLogin account?

Reply:
Yup, these were generated using the tool that is in the ``test/mocks/auth`` file.  OneLogin key uses a different signing algorithm as well.  Pretty sure tests would fail if that were used.